LunaMary/src/SelectedQuiz.java

```diff
@@ -0,0 +1,70 @@
+import java.util.List;
+
+
+class SelectedQuiz implements IQuestion {
+
+    private String title;
+    private List<Option> optionList;
+
+    public SelectedQuiz(String title, List<Option> optionList) {
+        this.title = title;
+        this.optionList = optionList;
+    }
+
+    public String getTitle() {
+        return title;
+    }
+
+    public void setTitle(String title) {
+        this.title = title;
+    }
+
+    public List<Option> getOptionList() {
+        return optionList;
+    }
+
+    public void setOptionList(List<Option> optionList) {
+        this.optionList = optionList;
+    }
+
+
+    public void answer(String resp) {
+
+    }
+
+    public String display() {
+
+        String result = "P1. " + getTitle() + "\n";
```

Comment:
This is a question that can be a part of many Quizzes. And in a Quiz this question can be either the first or the last question. That's why the "P1. " part of this shouldn't be here.